### PR TITLE
feat: define a `medibloc/panacea-core` BSR repository

### PR DESCRIPTION
**Pull request by youngjoon-lee**
### I WILL OPEN A WORKSHOP ABOUT [BUF](https://docs.buf.build/introduction). NO WORRIES. 

I defined a new `medibloc/panacea-core` BSR (Buf Schema Registry) repository: https://buf.build/medibloc/panacea-core, so that other Go projects (such as panacea-oracle) can import proto files defined in this repository. 

The Buf also has a [dependency management](https://docs.buf.build/tour/add-a-dependency), and Cosmos team already published their proto files in BSR: https://buf.build/cosmos/cosmos-sdk.
But unfortunately, their BSR repository is available only for Cosmos SDK v0.46+: https://github.com/cosmos/cosmos-sdk/tree/v0.47.0-rc2/proto#sdk-x-buf.
So, we should keep third-party proto files in this git repo until we upgrade Cosmos SDK to v0.46+. These third-party proto files are also pushed to our [BSR](https://buf.build/medibloc/panacea-core), but will be deleted once we use Cosmos SDK v0.46+.

### CODE REVIEW

**Comment by gyuguen** [comment]
Do all proto files need to exist in the proto folder to use this?
I ask because the third_party is all moved

**Comment by youngjoon-lee** [comment]
> Do all proto files need to exist in the proto folder to use this? I ask because the third_party is all moved

Yes. Basically, there's no `-I` flag for Buf: https://docs.buf.build/tour/configure-and-build#configure-buf. It assumes that all proto files are under the `proto/` directory where the `buf.yaml` is located.

**Review by gyuguen — APPROVED** [review]
LGTM

File: .github/workflows/proto-registry.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Buf-Push
+# Protobuf runs buf (https://buf.build/) push updated proto files to https://buf.build/cosmos/cosmos-sdk
```
[the remark this answers is not in the document]
Comment (youngjoon-lee) [reply .github/workflows/proto-registry.yml]: 👍

**Review by H4NLee — APPROVED** [review]
lgtm :+1:

**Pull request merged**